app/src/main/java/com/github/se/studybuddies/ui/todo/CreateToDo.kt

```diff
@@ -0,0 +1,82 @@
+package com.github.se.studybuddies.ui.todo
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.se.studybuddies.data.Location
+import com.github.se.studybuddies.data.todo.ToDo
+import com.github.se.studybuddies.data.todo.ToDoStatus
+import com.github.se.studybuddies.navigation.NavigationActions
+import com.github.se.studybuddies.viewModels.ToDoViewModel
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneId
+
+@Composable
+fun CreateToDo(todoViewModel: ToDoViewModel, navigationActions: NavigationActions) {
+  val titleState = remember { mutableStateOf("") }
+  val descriptionState = remember { mutableStateOf("") }
+  val assigneeState = remember { mutableStateOf("") }
+  val locationState = remember { mutableStateOf(Location(0.0, 0.0, "")) }
+  val onLocationChanged: (Location) -> Unit = { newLocation -> locationState.value = newLocation }
+  val locationQuery = remember { mutableStateOf("") }
+
+  val selectedDate = remember { mutableStateOf(LocalDate.now()) }
+  val isOpen = remember { mutableStateOf(false) }
+
+  Column(modifier = Modifier.fillMaxSize().testTag("createScreen")) {

```

Comment:
You couldn't know but with my PR on Solo_study_correction, I tried to set a generalized UI structure. Thus if you have time, it would be nice to look at the code of the other similar screen such as the timer (concerning the use of Scaffold and the Top app bar).

Reply:
I will reformat this once the firebase is functional